CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 3.5.1 (2015-10-28)
+
+* (mvz) Make Attribute respect suppressing comments
```

Comment:
Perhaps add other changes as well?

* Changed parser dependency
* Allow matches in `reek_of` for message, lines, context and source.
* Improved documentation
* Add logo!
* Deprecate AppConfiguration.from_map in favor of AppConfiguration.from_hash.